src/javascript/app/common/country_base.js

```diff
@@ -18,9 +18,15 @@ const isEuCountry = () => {
     );
 };
 
+const isFrance    = () => {
```

Comment:
better to make the naming more general in case we add more countries later, so something like hasCFD, and there's no need for the whitespace before `=`

Reply:
Addressed in updated code.